app/services/foreman/template_snapshot_service.rb

```diff
@@ -70,6 +70,7 @@ def define_host_params(host)
         "syspurpose_usage" => "Development/Test",
         "syspurpose_sla" => "Self-Support",
         "syspurpose_addons" => "first addon, second addon, third addon",
+        "subscription_manager" => "true",
```

Comment:
Perhaps I missed it, but for better coverage we should implement this in develop too.

Reply:
> Perhaps I missed it, but for better coverage we should implement this in develop too.

Already in: https://github.com/theforeman/foreman/blob/cd1ea006eb8b4bc7910e86cf2d182b1b7b63c45f/app/services/foreman/template_snapshot_service.rb#L78